charter.md

```diff
@@ -0,0 +1,55 @@
+
+Meeting time and agenda should be announced by the chair at least 48 hours in advance, and meeting minutes posted in the repository.
+
+The Interop Team's work is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+All contributions are licensed under the terms in [CONTRIBUTING.md](https://github.com/web-platform-tests/wpt/blob/master/CONTRIBUTING.md).
+
+Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs).
+
+## Scope
+
+The Interop Team's scope includes metrics designed to measure and improve the interoperability of the web platform. These metrics are published once consensus to do so has been reached.
+
+These metrics are expected to be comprised of:
+
+- Groups of automated tests that can be used to automatically compute an interoperability score for specific platform features in implementations.
+- Work to improve the interoperability of the platform that cannot be measured through test pass rates and is scored by consensus decision.
+
+The team is responsible for maintaining [Interop 2021](https://wpt.fyi/interop-2021), [2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and defining future interoperability efforts and metrics.
+
+If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.
+
+### Out of Scope
+
+This is not a venue for standardization work. Further, the team will avoid creating metrics which depend on significant standards work. However, this does not preclude the team from liaising with existing standards groups to help progress work which affects the interoperability of the platform, or the ability to test and measure interoperability of existing features (including features whose specification is determined to be ready-for-implementation but where there is yet to be any implementation).
+
+Browser features that don't affect web developers, like bookmarks, are out of scope.
+
+## Team Members
+
+The following organizations make up the Interop Team, in alphabetical order:
+
+* Apple
+* Bocoup
+* Google
+* Igalia
+* Microsoft
+* Mozilla
+
+## Joining and Leaving the Team
+
+The Interop Team is made up of organizations with a track record of contributions to the web-platform-tests project.
+
+Organizations may apply to join the team by contacting the chair. New members will be added with a consensus decision of existing members.
```

Comment:
There was some earlier discussion about whether we want the details of any discussions and/or vote to determine consensus to be public here. I think we merged it into the broader public/private discussion, but it may be worthwhile checking we are all okay with this being public, which per the current draft it would be.

Reply:
How about like this?

```suggestion
Organizations may apply to join the team by contacting the chair. New members will be added with a consensus decision of existing members. The deliberations are confidential to the existing members.
```

Reply:
Seems fine to me.

Reply:
Thanks, I'll apply the change.